pkg/config/manager.go

```diff
@@ -158,6 +159,10 @@ func (m *Manager) GetOauthClientsSecretName() string {
 	return "oauth-client-secrets"
 }
 
+func (m *Manager) GetAddonManagedApiServiceParametersSecretName() string {
+	return "addon-managed-api-service-parameters"
```

Comment:
hey @austincunningham any ways of not making this hardcoded? 
on edge the name of the secret is addon-managed-api-service-internal-parameters

Reply:
waiting for @makslion PR to merge https://github.com/integr8ly/integreatly-operator/pull/2679